doc/source/rllib-env.rst

```diff
@@ -69,9 +75,6 @@ For a full runnable code example using the custom environment API, see `custom_e
 
    The gym registry is not compatible with Ray. Instead, always use the registration flows documented above to ensure Ray workers can access the environment.
 
-Configuring Environments
```

Comment:
did you make sure this isnt linked anywhere in the docs?

Reply:
Oh it's just moved.